Gordon360/Services/NewsService.cs

```diff
@@ -201,9 +201,7 @@ public StudentNews DeleteNews(int newsID)
 
             if (newsItem.Image != null)
             {
-                string fileName = newsItem.SNID + ".jpg";
-                string imagePath = NewsUploadsPath + fileName;
-                ImageUtils.DeleteImage(imagePath);
+                ImageUtils.DeleteImage(newsItem.Image);
```

Comment:
```suggestion
bar imagePath = Path.Combine(_webHostEnvironment.ContentRootPath, "browseable", "uploads", Path.GetFileName(newsItem.Image));
ImageUtils.DeleteImage(imagePath);
```
Isn't newsItem.Image a URL, not a local file path? I don't think ImageUtils.DeleteImage would be able to delete it properly if it's a URL.

Reply:
I'm seeing that you changed ImageUtils.DeleteImage now. Technically this would work then, but I would avoid changing the central helper function just to fix an issue with your changes. It would probably be better to convert the URL in newsItem.Image to a local file path here rather than in the ImageUtils function.